### PR TITLE
[MRG] Add close_key to params dict to be able to modify it from outside

### LINKED ISSUE #4425: Disable closing raw plot window with escape key
Is it possible to disable the current behavior that the raw plot window can be closed with the escape key? I didn't find this mapping explicitly in the code, only for a selection dialog.

### PR DESCRIPTION
**Pull request by cbrnr**
Fixes #4425

### CODE REVIEW

File: mne/viz/raw.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
                   event_nums=event_nums, clipping=clipping, fig_proj=None,
                   first_time=first_time, added_label=list(), butterfly=False,
                   group_by=group_by, orig_inds=inds.copy(), decim=decim,
-                  data_picks=data_picks)
+                  data_picks=data_picks, close_key='escape')
```
Comment (Eric89GXL) [inline mne/viz/raw.py]: I thought we agreed not to expose this?

Comment (Eric89GXL) [inline mne/viz/raw.py]: ... oh this is in params, never mind!

**Comment by Eric89GXL** [comment]
To save travis some pain I killed all builds except the style one

**Comment by agramfort** [comment]
good to go @cbrnr ?

no objection on my side

**Comment by cbrnr** [comment]
Yes!

**Comment by Eric89GXL** [comment]
(AppVeyor should have caught it even though I cancelled most Travis builds...)

In any case, I've pushed a commit that should fix it

**Comment by agramfort** [comment]
sorry

**Comment by cbrnr** [comment]
Sorry about that - are there any other places where something like `plot_raw` is called?

**Comment by jaeilepp** [comment]
I think you could just set the ``close_key`` in ``_prepare_mne_browse_raw`` and it should work. And you probably want to do the same for ``_prepare_mne_browse_epochs`` too.

**Pull request merged**